js/script.js

```diff
@@ -0,0 +1,176 @@
+// variables and functions
+
```

Comment:
Keep making sure that all variables/functions are declared first. It'll be a habit that only helps you in the long run!